sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileSourceCodecSuite.scala

```diff
@@ -56,6 +56,7 @@ class ParquetCodecSuite extends FileSourceCodecSuite {
   override def format: String = "parquet"
   override val codecConfigName: String = SQLConf.PARQUET_COMPRESSION.key
   // Exclude "lzo" because it is GPL-licenced so not included in Hadoop.
+  // TODO: test "brotli" if the system is not of ARM64 CPU architecture.
```

Comment:
Ur, @gengliangwang . This is not a TODO item. Like `lzo`, I expect our decision why we exclude `brotli` here.

Reply:
Stephen casted -1 
- https://lists.apache.org/thread.html/rb92464f98a2b15f2a66a7f2c6bea5f422eabfe00e4de0a17d4f0a20b%40%3Cdev.spark.apache.org%3E
> I was going to -1 this because of the com.github.rdblue:brotli-codec:0.1.1 dependency, which is not available on Maven Central, and therefore is not available from our repository manager (Nexus).

Reply:
Got it. I misunderstood your point. I have updated the comment.